src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/tts_get_language_response.cc

```diff
@@ -91,6 +91,11 @@ void TTSGetLanguageResponse::Run() {
   event.raise(application_manager_.event_dispatcher());
 }
 
+void TTSGetLanguageResponse::onTimeOut() {
```

Comment:
What is the reason to create empty override?

Reply:
@LuxoftAKutsan i think you saw first commit, it was a preparation for further implementation, in next commit `onTimeOut` is not empty  [f64797](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/f64797882a55c7e3f0ec0787fbd4208136798d5f)